spec/bookingrequest_spec.rb

```diff
@@ -0,0 +1,18 @@
+describe BookingRequest do
+  describe '.create' do
+    it 'should add a booking request to the database' do
+      expect(BookingRequest.create(space_id: 1, date: '2022-05-17')).to be_an_instance_of(BookingRequest).and have_attributes(space_id: '1', date: '2022-05-17')
+    end
+  end
+
+  describe '.find' do
+    it "selects specific booking request by id" do
+      booking_request = BookingRequest.create(space_id: 1, date: '2022-05-17')
+      found_booking_request = BookingRequest.find(id: booking_request.id)
+      
+      expect(found_booking_request.id).to eq(booking_request.id)
```

Comment:
Refactor 13-15 to one line, using be_an_instance_of and have_attributes (same structure as line 4 in this file)